lib/tasks/link_checker.rake

```diff
@@ -16,8 +16,7 @@ def gather_urls object
   urls
 end
 
-desc "link checker"
-task :link_checker, [:date] => [:environment] do |_, _args|
+def get_urls_from_locale_files
```

Comment:
It would be great if unit tests could be added for this method and `gather_urls` above.